Makefile

```diff
@@ -88,7 +88,11 @@ venv:
 # install SLSA verifier binary, download mvnw, and gradlew.
 .PHONY: setup
 setup: force-upgrade setup-go setup-binaries setup-schemastore
+ifeq ($(PRE_COMMIT_HOOKS),false)
```

Comment:
This is purely for dev convenience & is also related to the 3rd party(Souffle) build break.

`make setup` triggers the `pre-commit` hooks and because tests fail (as expected), `build`/`setup` step is blocked as a result.